web/index.html

```diff
@@ -131,13 +131,23 @@
     <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, interactive-widget=resizes-content">
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
     <link rel="shortcut icon" id="favicon" href="/favicon.png">
-    <% if (htmlWebpackPlugin.options.usePolyfillIO) { %>
+    <% if (htmlWebpackPlugin.options.isWeb) { %>
         <!-- polyfill.io is only needed on Web to support older browsers. It should not be loaded for desktop -->
         <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CResizeObserver&flags=gated"></script>
+        <% if (htmlWebpackPlugin.options.isProduction) { %>
+            <!-- Google Tag Manager will only be loaded on web for now -->
```

Comment:
Can you update the comment to include the link where this snippet came from? Is it [here](https://tagmanager.google.com/)?

Also, I think saying that it's "only loaded on web" inside `web/index.html` is a bit redundant. Even if we add the Google tag on native platforms, it would still be here in `web/index.html`.

Reply:
Answered where the ID comes from in the internal discussion. Will remove the comment.